techniques/client-side-script/SCR40.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Using the CSS prefers-reduced-motion query in JavaScript to prevent motion</title>
+  <link rel="stylesheet" type="text/css" href="../../css/editors.css">
+</head>
+<body>
+
+<h1>Using the <abbr title="Cascading Style Sheets">CSS</abbr> <code>prefers-reduced-motion</code> query in JavaScript to prevent motion</h1>
+
+<section id="meta">
+  <h2>Metadata</h2>
+  <p id="id"></p>
+  <p id="technology"></p>
+  <p id="type"></p>
+</section>
+
+<section id="applicability">
+  <h2>When to Use</h2>
+  <p>
+    JavaScript animation which causes motion that is triggered by user interactions.
+  </p>
+</section>
+
+<section id="description">
+  <h2>Description</h2>
+  <p>The objective of this technique is to allow users to prevent animations (including motion animations) from being displayed on web pages, by using JavaScript to evaluate the <code class="language-css">prefers-reduced-motion</code> CSS Media Query.</p>
+  <p>Some users experience distraction or nausea from animated content. For example, if scrolling a page causes elements to move (other than the essential movement associated with scrolling the content, which is under the user's control) it can trigger vestibular disorders.</p>
+  <p>Media queries that selectively enable/disable JavaScript-driven animations based on operating system or user agent preferences allow users to prevent those symptoms.</p>
+  <p>The understanding document for <a href="../../Understanding/animation-from-interactions.html#resources">Animation from Interactions</a> includes links for changing the 'reduced motion' setting.</p>
+</section>
+
+<section id="examples">
+  <h2>Examples</h2>
+  <section class="example">
+    <h3>Evaluating the <code>prefers-reduced-motion</code> CSS Media Query in JavaScript</h3>
+    <p>Users can indicate their motion preference for interfaces in their system. This choice can be detected in JavaScript by evaluating the <code>prefers-reduced-motion</code> CSS Media Query. The script can then decide to enable or disable animation effects based on the result of the media query test.</p>
+<pre xml:space="preserve"><code class="language-js">const mediaQueryList = window.matchMedia("(prefers-reduced-motion: no-preference)");
+
+if (mediaQueryList.matches) {
+  /* The user has not expressed a preference for reduced motion – run JavaScript-based animation */
+}
+</code></pre>
+  </section>
+</section>
+
+<section id="tests">
+  <h2>Tests</h2>
+  <section class="test-procedure">
+    <h3>Procedure</h3>
+    <p>For each interactive element that triggers a motion animation as a result of a user interaction:</p>
+    <ol>
+      <li>Enable the reduced motion setting in your system;</li>
+      <li>Check that the motion animation is not essential;</li>
+      <li>Check that the motion animation is suppressed.</li>
+    </ol>
+  </section>
+  <section class="test-results">
+    <h3>Expected Results</h3>
+    <ul>
+      <li>#2 and #3 are true.</li>
```

Comment:
I think it's a bit confusing to include the "not essential" item; if the technique's expected results demands that the user verify that #3 is true, it doesn't matter (for the purposes of the SC) whether or not the motion animation is essential, so there's no need to check it. I think if we want to include an *out* for essential animation, you'd want something more like:

```suggestion
      <li>Enable the reduced motion setting in your system;</li>
      <li>Check that the motion animation is essential;</li>
      <li>Check that the motion animation is suppressed.</li>
    </ol>
  </section>
  <section class="test-results">
    <h3>Expected Results</h3>
    <ul>
      <li>#2 or #3 is true.</li>
```

(I do see that this is the same as the previous technique; if we agree to change it, we should change it consistently for both techniques)